feat: Add Debian-specific information to LVGL OOB docs (#201)
TexasInstruments/processor-sdk-doc: source/system/Demo_User_Guides/TI_LVGL_Demo_User_Guide.rst

Launching the TI LVGL Demo
**************************

.. ifconfig:: CONFIG_sdk in ('SITARA')

>     The demo will auto launch upon Linux booting on the EVM. Follow the below instructions to flash the SD card:

Reviewer: Incorrect indentation depth

    1. Flash an SD card with the :file:`tisdk-default-image`. User can download the :file:`tisdk-default-image` wic image from |__SDK_DOWNLOAD_URL__|.
    Please follow the instructions from here to :ref:`Flash an SD card <processor-sdk-linux-create-sd-card>`.

>     2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power on the EVM.

Reviewer: Mentioning the device specific jumper wire in a device agnostic section. Please clarify that this is only required if mentioned previously.

    The TI LVGL Demo will launch automatically when the device is fully booted.

.. ifconfig:: CONFIG_sdk in ('DebianSDK')

   Upon booting the EVM, Weston is launched automatically. To launch the LVGL OOB demo, follow the following
   instructions:

    1. Flash an SD card with the :file:`tisdk-debian-trixie` wic image. User can download the wic image from |__SDK_DOWNLOAD_URL__|.
    Please follow the instructions from here to :ref:`Flash an SD card <processor-sdk-debian-create-sd-card>`.

    2. Insert the flashed SD card to the board, connect the display, mouse/touch-input, ethernet cable, aux cable, jumper wire and power the EVM on.

>     3. Weston is launched on boot. Shut it down with `systemctl stop weston`.

Reviewer 2: ```suggestion
    3. Weston wallpaper is launched on boot. You can stop it by running :code:`systemctl stop weston`.
```

Reviewer: Semantics, but I would say weston is launched and the wallpaper is displayed.

>     4. Lauch the demo by typing `lvglsim` into the UART command prompt.

Reviewer 2: ```suggestion
    4. Lauch the demo by typing :code:`/usr/bin/lvglsim` into the UART command prompt.
```

It's a good practice to mention absolute path

      By default CC33xx is configured at boot, so here are the steps that can be followed to enable it:

      .. ifconfig:: CONFIG_sdk in ('SITARA')

>           .. code-block:: console

Reviewer: Incorrect indentation depth. A lot of this file is mixed indentation depths.

>              $ systemctl stop ti-lvgl-demo

Reviewer 2: ```suggestion
             root@<machine>:~ systemctl stop ti-lvgl-demo
```

For all references

             $ cd /usr/share/cc33xx
             $ ./sta_start.sh
             $ ./sta_connect.sh -s WPA-PSK -n <SSID> -p <PASSWORD>
             $ udhcpc -i wlan0
             $ systemctl start ti-lvgl-demo

>       .. ifconfig:: CONFIG_sdk in ('DebianSDK')

Reviewer: Incorrect indentation depth

Building the TI LVGL Demo from Sources
**************************************

.. ifconfig:: CONFIG_sdk in ('SITARA')

    The TI LVGL Demo is enabled in :file:`tisdk-default-image` yocto filesystem for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that
    the binary itself does not have asset images and slides built in it. :file:`tisdk-default-image` contains the required assets within
    :file:`/usr/share/ti-lvgl-demo/`. Place any additional assets here while making any modifications. Yocto recipe for
    building this demo can be found at
    `github: ti-lvgl-demo.bb <https://github.com/TexasInstruments/meta-tisdk/blob/scarthgap/recipes-demos/ti-lvgl-demo/ti-lvgl-demo.bb>`__

.. ifconfig:: CONFIG_sdk in ('DebianSDK')

>     The TI LVGL Demo is installed in :file:`tisdk-debian-trixie` wic image for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that

Reviewer 2: ```suggestion
    The TI LVGL Demo is packaged in :file:`tisdk-debian-trixie` wic image for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that
```
